Tests/SwiftCliCoreTests/ChartConstructorTest.swift

```diff
@@ -95,14 +117,84 @@ class FullChartPrinterTests: XCTestCase {
         XCTAssertEqual(result, expectedresult)
     }
     func testFullPatternTwoRow() throws {
-        let result = ChartConstructor().make_chart(stitch_array: [["k1", "k1", "p1", "p1", "k1", "k1"],["k1", "k1", "p1", "p1", "k1", "k1"]])
+        let result = ChartConstructor().make_chart(stitch_array: [["k1", "k1", "p1", "p1", "k1", "k1"],["p1", "p1", "k1", "k1", "p1", "p1"]])
         let expectedresult = """
 ┌─┬─┬─┬─┬─┬─┐
-│ │ │-│-│ │ │
+│-│-│ │ │-│-│
 ├─┼─┼─┼─┼─┼─┤
 │ │ │-│-│ │ │
 └─┴─┴─┴─┴─┴─┘
 """
         XCTAssertEqual(result, expectedresult)
     }
+    
+    func testFullPatternRightSSKDecrease() throws {
+        let result = ChartConstructor().make_chart(stitch_array: [["k1", "k1", "p1", "p1", "k1", "k1", "p1", "k1"],["k1", "k1", "p1", "p1", "k1", "k1", "ssk"],["k1", "k1", "p1", "p1", "k1", "ssk"]])
+        let expectedresult = """
+┌─┬─┬─┬─┬─┬─┐
+│ │ │-│-│ │\\│
+├─┼─┼─┼─┼─┼─┼─┐
+│ │ │-│-│ │ │\\│
+├─┼─┼─┼─┼─┼─┼─┼─┐
+│ │ │-│-│ │ │-│ │
+└─┴─┴─┴─┴─┴─┴─┴─┘
+"""
+        XCTAssertEqual(result, expectedresult)
+    }
+    
+    
+    
+    func testFullPatternTripleRightSSKDecrease() throws {
+        let result = ChartConstructor().make_chart(stitch_array: [["k1", "k1", "p1", "p1", "k1", "k1","k1", "k1","k1", "k1"],["k1", "k1", "p1", "p1", "ssk", "ssk", "ssk"]])
+        let expectedresult = """
+┌─┬─┬─┬─┬─┬─┬─┐
+│ │ │-│-│\\│\\│\\│
+├─┼─┼─┼─┼─┼─┼─┼─┬─┬─┐
+│ │ │-│-│ │ │ │ │ │ │
+└─┴─┴─┴─┴─┴─┴─┴─┴─┴─┘
+"""
+        XCTAssertEqual(result, expectedresult)
+    }
+//
```

Comment:
Why the following tests commented out?

Reply:
The commented-out following tests were from last week, when I got ahead of myself and made more than one test at once. That was a mistake, I forgot about the principles of red-green TDD when I did that. I deleted them and will rewrite them when they become necessary.